x-pack/solutions/observability/packages/kbn-evals-suite-obs-ai-assistant/evals/knowledge_base/knowledge_base.spec.ts

```diff
@@ -0,0 +1,199 @@
+          },
+        ],
+      });
+      await evaluateKnowledgeBase({
+        dataset: {
+          name: 'kb_functions_recall',
+          description: 'Tests the recall functions of the knowledge base',
+          examples: [
+            {
+              input: {
+                question: `What is this cluster used for?`,
+              },
+              output: {
+                criteria: [
+                  'Calls the "context" function to respond to What is this cluster used for?',
+                  'Effectively recalls that this cluster is used to test the AI Assistant using the Observability AI Evaluation Framework',
+                ],
+              },
+              metadata: {},
+            },
+          ],
+        },
+      });
+    });
+  });
+
+  // --- Tests for kb retrieval ---
+  evaluate.describe('kb retrieval', () => {
+    const testDocs = [
+      {
+        id: 'acme_teams',
+        title: 'ACME DevOps Team Structure',
+        text: 'ACME maintains three primary DevOps teams: Platform Infrastructure (responsible for cloud infrastructure and Kubernetes clusters), Application Operations (responsible for application deployments and monitoring), and Security Operations (responsible for security monitoring and compliance). Each team maintains a separate on-call rotation accessible via PagerDuty. The current on-call schedule is available in the #oncall Slack channel or through the PagerDuty integration in Kibana.',
+      },
+      {
+        id: 'acme_monitoring',
+        title: 'Alert Thresholds',
+        text: 'Standard alert thresholds for ACME services are: API response time > 500ms (warning) or > 1s (critical), error rate > 1% (warning) or > 5% (critical), CPU usage > 80% (warning) or > 90% (critical), memory usage > 85% (warning) or > 95% (critical). Custom thresholds for specific services are documented in the service runbooks stored in Confluence under the "Service Specifications" space.',
+      },
+      {
+        id: 'acme_infra',
+        title: 'Database Infrastructure',
+        text: 'Primary transactional data is stored in PostgreSQL clusters with read replicas in each region. Customer metadata is stored in MongoDB with M40 clusters in each region. Caching layer uses Redis Enterprise Cloud with 15GB instances. All database metrics are collected via Metricbeat with custom dashboards available under "ACME Databases" in Kibana. Database performance alerts are configured to notify the DBA team via the #db-alerts Slack channel.',
+      },
+    ];
+
+    evaluate.beforeAll(async ({ knowledgeBaseClient, esClient }) => {
+      await knowledgeBaseClient.importEntries({ entries: testDocs });
+    });
+
+    evaluate.afterAll(async ({ esClient }) => {
+      await esClient.deleteByQuery({
+        index: '.kibana-observability-ai-assistant-kb-*',
+        ignore_unavailable: true,
+        query: {
+          match: {
+            text: 'ACME',
+          },
```

Comment:
I don't think you need to limit to entries matching `ACME`. Can you make a helper `clearKnowledgeBase` similar to what we have for the API tests?